content/en/data_jobs/airflow.md

```diff
@@ -71,58 +71,72 @@ In Datadog, view the [Data Jobs Monitoring][2] page to see a list of your Airflo
 {{% tab "Amazon MWAA" %}}
 ## Requirements
 
-* [Apache Airflow 2.9.0][1] or later
-* [apache-airflow-providers-openlineage 1.11.0][2] or later
+* [Apache Airflow 2.5.0][1] or later
+* [apache-airflow-providers-openlineage][2] or [openlineage-airflow][10] depending on your Airflow version
 
 ## Setup
 
-Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
-
 To get started, follow the instructions below.
 
 1. Install `openlineage` provider by adding the following into your `requirements.txt` file:
 
-   ```text
-   apache-airflow-providers-openlineage>=1.11.0
-   ```
+    For **Airflow 2.7 or later**:
 
-   Ensure the openlineage provider version is compatible with your constraints file. If no constraints file is specified in `requirements.txt`, ensure compatibility with the [default Apache Airflow constraints][8] for your Airflow version. Refer to the [Amazon MWAA User Guide][7] for guidance on specifying Python dependencies in `requirements.txt`.
+      ```text
+      apache-airflow-providers-openlineage
+      ```
+
+    For **Airflow 2.5 & 2.6** :
+
+      ```text
+      openlineage-airflow
+      ```
 
 2. Configure `openlineage` provider. The simplest option is to set the following environment variables in your [Amazon MWAA start script][3]:
 
    ```shell
    #!/bin/sh
-
    export OPENLINEAGE_URL=<DD_DATA_OBSERVABILITY_INTAKE>
    export OPENLINEAGE_API_KEY=<DD_API_KEY>
+   # Optional, allows Datadog to tag the jobs with the env name
```

Comment:
Let's not call this optional. Even if it will work without, it will benefit all users to just set a value here.